helm/templates/dcgm-exporter-role.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: "{{ template "dcgm-exporter.name" . }}-role"
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "amazon-cloudwatch-observability.labels" . | nindent 4}}
+rules:
+- apiGroups: [""]
+  resources: ["configmaps"]
+  resourceNames: ["{{ .Release.Namespace }}:{{ .Values.dcgmExporter.configmap }}"]
```

Comment:
Does this work when its namespace:configMap? I would have expected this to just be configMap.